radiography/src/main/java/radiography/ViewFilter.kt

```diff
@@ -0,0 +1,17 @@
+package radiography
+
+import android.view.View
+
+/**
+ * Used to filter out views from the output of [Radiography.scan].
+ */
+interface ViewFilter {
```

Comment:
Should we include some sort of helper (maybe an operator?) to combine filters?

Reply:
oh yeah totally forgot I mean to do that.

Reply:
```
  /**
   * Creates a new filter that combines this filter with [otherFilter]
   */
  infix fun and(otherFilter: ViewFilter): ViewFilter {
    val thisFilter = this
    return object : ViewFilter {
      override fun matches(view: View) = thisFilter.matches(view) &&
          otherFilter.matches(view)
    }
  }
```